openssh/test/lib/habitat.bash

```diff
@@ -0,0 +1,118 @@
+habitat::info() {
```

Comment:
Is this file needed, given the instructions in the README work?

I think it would be better to have a common `test_hab_plan` function scripted inside the the [bin directory](https://github.com/habitat-sh/core-plans/tree/master/bin)

Reply:
Added a task to the backlog for adding test_hab_plan to bin directory, to be reviewed again later.  See https://github.com/orgs/habitat-core-plans/projects/1#card-19498245